README.md

```diff
@@ -1,36 +1,65 @@
 # H2
-Inspired by the Helium app, **H2** is a minimalist browser to watch embeded videos and more in  picture-in-picture mode.
+
+Inspired by the Helium app, **H2** is a minimalist browser to watch embeded videos and more in picture-in-picture mode.
+
 # Features
+
 :tv: Works with any Youtube URL.
 
 :scroll: Works with PDFs links, Google Docs URLs.
 
 :computer: Works on all platforms, thanks to electron ❤️
 
-:clock9:[WIP] After [#22](https://github.com/poush/H2/pull/22) a basic image editor, many other features coming soon.
+:art: Image editor supports contrast, brightness, hue, sepia and save as PNG.
+
+:clock9:[WIP] Many other features coming soon.
+
 # Example
+
 ![demo](img/demo.gif)
+
 ## Installation
-* [Windows](docs/windows.md)
-* [Mac](docs/mac.md)
-* [Linux](docs/linux.md)
+
+- [Windows](docs/windows.md)
+- [Mac](docs/mac.md)
+- [Linux](docs/linux.md)
+
 ## Usage
-* Copy any youtube, PDF or google docs URL and press `Ctrl/Command + Shift + V`
-* [Keyboards Shortcuts](docs/shortcuts.md)
+
+- Copy any youtube, PDF or google docs URL and press `Ctrl/Command + Shift + V`
+- [Keyboards Shortcuts](docs/shortcuts.md)
+
 ## Contributing
+
 ```bash
 # Clone this repository
+
 git clone https://github.com/poush/h2
+
 # Go into the repository
+
 cd h2
+
 # Install dependencies
+
 npm install
+
 # Run the app
+
 npm start
 ```
+
 ## Bug
+
 [Known Bugs](https://github.com/poush/H2/issues?q=is%3Aopen+is%3Aissue+label%3Abug)
+
 ## Credits
 
+| Contributor                                                               |       Name |                                   |
```

Comment:
I would prefer this to be present is a separate file like "contributors.md" which can be linked here.